extract.py

```diff
@@ -20,8 +20,16 @@ def main():
         tb = get_text_box_from_xml_element(textbox)
         for text_line in tb.text_lines:
```

Comment:
This might be premature, but..

looking at the TextBox class, it feels the lines are the natural thing to be iterating over. I'd be tempted to make TextBox iterable. Since the lines are in a list which is already iterable, this is easier than  https://docs.python.org/3.7/tutorial/classes.html#iterators makes it seem. https://docs.python.org/3.7/library/stdtypes.html#iterator-types describes this better.

````python
class TextBox():

    def __iter__(self):
        return self.text_lines
````

Then you can just:

````python
for line in tb:
````